### PR TITLE
CO-2848 sms_sponsorship migration v11

### PR DESCRIPTION
**Pull request by josueBulle**
depends on others pull requests

crm_compassion -> crm_request -> crm_claim_type

### CODE REVIEW

**Comment by ecino** [comment]
flake8 issues (alignment) :
``` 
sms_sponsorship/models/sms_child_request.py:241:17: E122 continuation line missing indentation or outdented
sms_sponsorship/models/sms_child_request.py:242:17: E122 continuation line missing indentation or outdented
sms_sponsorship/models/sms_child_request.py:243:17: E122 continuation line missing indentation or outdented
sms_sponsorship/models/sms_child_request.py:244:17: E122 continuation line missing indentation or outdented
sms_sponsorship/models/sms_child_request.py:245:17: E122 continuation line missing indentation or outdented
sms_sponsorship/models/sms_child_request.py:246:17: E122 continuation line missing indentation or outdented
sms_sponsorship/models/sms_child_request.py:247:17: E122 continuation line missing indentation or outdented
sms_sponsorship/models/sms_child_request.py:248:17: E122 continuation line missing indentation or outdented
sms_sponsorship/models/sms_child_request.py:249:17: E122 continuation line missing indentation or outdented
sms_sponsorship/models/sms_child_request.py:250:13: E122 continuation line missing indentation or outdented
```

File: sms_sponsorship/models/sms_child_request.py [hunk 1]
```diff
@@ -34,8 +32,7 @@ class SmsChildRequest(models.Model):
     _rec_name = 'child_id'
     _order = 'date desc'
 
-    sender = Phone(partner_field='partner_id',
-                   country_field='country_id')
+    sender = fields.Char('Phone')
```
Comment (ecino) [inline sms_sponsorship/models/sms_child_request.py:36]: The model should inherit from `phone.validation.mixin` and add a property `_phone_name_fields = ['sender']`

**Review by ecino — CHANGES_REQUESTED** [review]

**Review by ecino — CHANGES_REQUESTED** [review]
You have places in the code where you still access `staff.notification.settings`. Please replace it by `res.config.settings` to avoid errors. I don't know if it's as well related to your changes, but the Compassion settings view is broken:
image

**Comment by josueBulle** [comment]
> I don't know if it's as well related to your changes, but the Compassion settings view is broken:

This field is defined in the OCA module "base_phone" and isn't related to this PR
(Note that it is present in all the sections, even with sms_sponsorship uninstalled)

**Review by ecino — APPROVED** [review]

**Pull request merged**